SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -51,7 +51,7 @@ def or {w : Nat} (x y : IntW w)  (flag : DisjointFlag := {disjoint := false}) :
   let y' ← y
```

Comment:
can you put `(x y : IntW w)` into curly brackets? I could not suggest a comment on that line in the review :(

Reply:
Aren't x and y supposed to be explicit arguments here?